linkerd/telemeter/usage/src/main/scala/io/buoyant/linkerd/telemeter/UsageDataTelemeter.scala

```diff
@@ -163,10 +162,13 @@ class UsageDataTelemeter(
   log.info(s"connecting to usageData proxy at $metricsDst")
   val client = Client(metricsService, config, pid, orgId)
 
-  val adminHandlers = Seq(
-    Handler("/admin/metrics/usage", new UsageDataHandler(metricsService, config, pid, orgId, registry))
+  override def adminHandlers: Seq[Handler] = Seq(
+    Handler("/admin/metrics/usage", new UsageDataHandler(metricsService, config, pid, orgId, registry)),
+    Handler("/usage", new UsageDataAdminHandler)
```

Comment:
i considered passing `UsageDataHandler` into `UsageDataAdminHandler` and generating the usage json payload server-side, but i think leveraging the templating/js infrastructure on the client-side makes more sense for this use case.